mundane-bytes/src/main/scala/com/ambiata/mundane/bytes/Buffer.scala

```diff
@@ -0,0 +1,26 @@
+package com.ambiata.mundane.bytes
+
+case class Buffer(bytes: Array[Byte], offset: Int, var length: Int) {
+
+  /** Ensure there is enough space in the array, and if not return a _new_ [[Buffer]] and array with the original values */
+  def allocate(i: Int): Buffer =
```

Comment:
Stylistic comment only, so ignore if you want, but would you consider moving this to the companion?